velox/expression/CastExpr-inl.h

```diff
@@ -178,7 +178,7 @@ void CastExpr::applyToSelectedNoThrowLocal(
 /// @param row The index of the current row
 /// @param input The input vector (of type FromKind)
 /// @param result The output vector (of type ToKind)
-template <TypeKind ToKind, TypeKind FromKind, bool Truncate, bool LegacyCast>
+template <TypeKind ToKind, TypeKind FromKind, bool LegacyCast>
```

Comment:
LegacyCast template parameter applies only to Presto, no? Should we remove it? CC: @gggrace14